pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -1987,6 +1988,7 @@ func scheduleReboot(reboot *zconfig.DeviceOpsCmd,
 					err)
 			}
 		}
+		getconfigCtx.zedagentCtx.rebootConfigCounter = reboot.Counter
```

Comment:
This makes no sense to me. You only report a non-zero value after a rebootCommand has been processed?

You need to have zedagent parse the /config file on starting to set rebootConfigCounter, and then update it when the file gets updated (by this code).

Reply:
Ah.. yes.. missed that case.. Fixed it.